src/Orchard.Web/Modules/Orchard.Email/Controllers/EmailAdminController.cs

```diff
@@ -12,12 +11,12 @@
 namespace Orchard.Email.Controllers {
     [Admin]
     public class EmailAdminController : Controller {
-        private readonly ISmtpChannel _smtpChannel;
-        private readonly IOrchardServices _orchardServices;
+        private readonly ISmtpChannel smtpChannel;
```

Comment:
Can you revert all the names you changed?

Reply:
@sebastienros Okay. Since it is naming convention that Orchard team use, I will follow the standard.